spec/services/google/search_service_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/http_client_helpers'
+
+RSpec.describe Google::SearchService, type: :service do
+  describe '#search!' do
+    context 'when search a keyword' do
+      it 'perform a request against www.google.com' do
```

Comment:
Again, some issues with the usage of context here. For consistency, prefer using `given` (instead of mixing `given` and `when` in the same codebase). Here there should be two contexts: `given a successful request` and `given a failed request`

Reply:
Fixed in df8db9fa717be526c4f37e066016095bc23c30e6